models/nc-news-models.js

```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection")
+
+exports.fetchTopics = () => {
+    return db
+    .query("SELECT * FROM topics;")
+    .then(({rows}) => {
+        if(!rows.length) {
+            return Promise.reject({status: 404, msg: "Not Found"})
+        }
```

Comment:
Two questions for you....
1. Do you need this error? Generally a 404 is returned when we've requested a specific record or a subset of records in the database (E.g. I want all the coffee related topics, but there are no coffee topics! ☕ ). But here we just want all the topics from the database.
2. Where does this error go 👀  You have not caught it anywhere, and you don't have any tests for it either...

Reply:
Thank;s for spotting this. I actually had it caught, passed and handled. But then I realised, that with this endpoint it makes no sense and I ended up deleting it and obviously I forgot to remove it from the model. Will do :) and I will do my best to remember to get rid of comments ;)